settings/nitrofiles/languages/english.ini

```diff
@@ -69,6 +70,7 @@ RELEASE = Release
 NIGHTLY = Nightly
 
 DESCRIPTION_LANGUAGE_1 = Avoid the limited selections of your console language by setting this option.
+DESCRIPTION_TITLELANGUAGE_1 = Set Banner Title Language for DS ROMs expect DSiWare.
```

Comment:
You should put "except DSiWare" in a parenthesis (ie: `(except DSiWare)`)

Reply:
Wait me a moment, ha.